CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Features
 - Update the single-family detached and multifamily projects with more up-to-date lighting stock distributions ([#392](https://github.com/NREL/OpenStudio-BuildStock/pull/392))
 - Update Insulation Finished Attic tsv with more options for insulation levels ([#395](https://github.com/NREL/OpenStudio-BuildStock/pull/395))
 - Add ability to ignore comment lines with the "#" symbol ([#408](https://github.com/NREL/OpenStudio-BuildStock/pull/408))
+- New geospatial characteristics have been added or updated. New geospatial caracteristics are as follows: ASHRAE IECC Climate Zone 2004, State, County, PUMA, Census Division, Census Region, Building America Climate Zone, and ISO/RTO Region. ([#416](https://github.com/NREL/OpenStudio-BuildStock/pull/416)).
```

Comment:
"characteristics"

Reply:
I can make these changes. Thanks.

Reply:
Can you mention that TSV order has changed (now starts with ASHRAE IECC zone)?

Reply:
@joseph-robertson: I just remembered why we need both Geometry Building Type ACS and Geometry Building Type RECS. Geometry Building Type ACS is needed as a dependency in Geometry Building Number Units MF. There are only certain combinations of unit number and stories that our ResidentialGeometryCreateMultifamily measure can handle, and therefore need the more detailed number of units specified in Geometry Building Type ACS.

Reply:
Suggest adding something like "Now using data from the American Community Survey Public Use Microdata Sample (ACS PUMS) for Building Type, Vintage, and Heating Fuel."